apps/storybook.namekit.io/stories/Namekit/TruncatedText.stories.tsx

```diff
@@ -48,54 +40,59 @@ type Story = StoryObj<typeof TruncatedText>;
 
 export const ShortText: Story = {
   args: {
-    text: getExampleTruncatedText(TruncatedTextExample.SHORT_TEXT),
+    text: SHORT_TEXT,
+    maxDisplayWidth: 300,
```

Comment:
It's important to add the explicit `maxDisplayWidth` param here. It's not a default for the overall UI component.